TACC/Core-Portal
# 737: Use `_common` `<Button>` component in View Team button

## Overview: ##

Use our new button component (and [new button design]) for View Team button (for #636).

[^1]: I gave Design a [internal inquiry (different use case)](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1653600337362999) and a [private inquiry (this use case)](https://tacc-team.slack.com/archives/G01K9ET7JM7/p1653604140397549).

<details><summary>To Do</summary>

- [x] try more padding for buttons ([source](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1654877292004179?thread_ts=1654815547.469369&cid=CQUA4D5KJ)) (and then use small button)
    - https://github.com/TACC/Core-Styles/pull/30
    - https://github.com/TACC/Core-Styles/commit/eafe102
    - https://github.com/TACC/Core-Portal/pull/649#commits-pushed-759126d
- [x] release & use core-styles `0.5.2` ([comment](https://github.com/TACC/Core-Portal/pull/649/files/18ef0d175261d6727cab909817b1d58cdcb87e7d#r896014931))
    - c63eac7

</details>

## Related Jira tickets: ##

* [FP-1635](https://jira.tacc.utexas.edu/browse/FP-1635)
* requires https://github.com/TACC/Core-Portal/pull/636

## Summary of Changes: ##

- use `_common/Button` component

## Testing Steps: ##

1. Open Allocations section.
2. Confirm that gray button has become standard secondary (white) button.
3. Scroll Allocations table. — 2022-06-13
4. Confirm button does **not** appear atop table head. — 2022-06-13

## UI Photos:

<details open><summary>New</summary>

| _new_ small[^2] | button underneath `<th>`
| - | - |
| ![View Button New Small Button](https://user-images.githubusercontent.com/62723358/173400614-247e0bac-02ee-4af5-a016-11ee0ce7aed4.png) | ![View Team Button Scroll](https://user-images.githubusercontent.com/62723358/173401136-7914e0bb-5b00-40a1-9218-492e6f967a9a.png) |

</details>

<details><summary>Options</summary>

| medium | small | _new_ small[^2] |
| - | - | - |
| <img width="616" alt="Gray Button (New Design)" src="https://user-images.githubusercontent.com/62723358/172958071-612c299a-3caf-4cd6-957a-bb94290f7922.png"> | <img width="616" alt="Gray Button (New Design) Small" src="https://user-images.githubusercontent.com/62723358/172958492-1cbec626-eba9-45b1-8ca3-134af2b59f4c.png"> | pending&nbsp;code |

</details>
<details><summary>History of UI</summary>

| before | [recent section design] | w/ bad button design[^1] | w/ [new button design] |
| - | - | - | - |
| ![Gray Button (Current App)](https://user-images.githubusercontent.com/62723358/170590851-b32e2456-0b30-45fb-a92a-b96e340b553b.png) | ![Gray Button (Design)](https://user-images.githubusercontent.com/62723358/170590235-4b46ed3d-08fa-47a0-9228-022f07f965c0.png) | ![White Button (New Button UI)](https://user-images.githubusercontent.com/62723358/170590240-a65308bd-3b64-4dda-bfd1-58f4098496a8.png) | <img width="616" alt="Gray Button (New Design)" src="https://user-images.githubusercontent.com/62723358/172958071-612c299a-3caf-4cd6-957a-bb94290f7922.png"> |

</details>

[recent section design]: https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/209a0ee5-8add-4931-955e-e33f20bdc9e5
[new button design]: https://xd.adobe.com/view/b37b291a-7798-4814-90f3-7ac63c48a3e0-e628/screen/d7a72466-4ae8-43fd-9531-44e033245ddb

[^1]: The first [new button design] (design doc since updated) had white secondary button. Design realized this wouldn't jive with Portal's existing design.

[^2]: Design got asked to make a choice or tweak. M.S. suggested small button with more padding.
